book/src/secrets-and-credentials-storage-class.md

```diff
@@ -112,6 +112,9 @@ The values of these parameters may be "templates". The `external-provisioner` wi
   * `${pvc.name}`
     * Replaced with the name of the `PersistentVolumeClaim` object that triggered provisioning.
     * Support added in CSI `external-provisioner` v1.2.0+
+  * `${pvc.annotations['<ANNOTATION_KEY>']}` (e.g. `${pvc.annotations['example.com/key']}`)
+    * Replaced with the value of the specified annotation from the `PersistentVolumeClaim` object that triggered provisioning
+    * Support added in CSI `external-provisioner` v4.0.2+
```

Comment:
I don't think there is a v4.0.2 release, the external-provisioned moved from v4.0.1 to v5.0.0.

Reply:
At the time of creating this PR, it was v4.0.1. Now, it needs to be updated. If someone is interested, they can pick it up